packages/cli/src/workflows/workflow.service.ts

```diff
@@ -137,6 +164,27 @@ export class WorkflowService {
 		});
 	}
 
+	private addWorkflowsCount(workflowsAndFolders: WorkflowAndFolderUnionFull[]) {
+		workflowsAndFolders.forEach((wf) => {
+			if (isFolder(wf)) {
+				wf.workflowsCount = wf?.workflows?.length ?? 0;
```

Comment:
`wf` should be alwasy defined

```suggestion
				wf.workflowsCount = wf.workflows?.length ?? 0;
```